app/web/package.json

```diff
@@ -110,5 +111,8 @@
     "timezone-mock": "^1.3.2",
     "typescript": "~5.7.2",
     "whatwg-fetch": "^3.6.20"
+  },
+  "resolutions": {
+    "@types/react": "^18.3.1"
```

Comment:
I added this as `react-simple-maps`'s  sub-dependencies are pulling in a conflicting version of @types/react (v19). 

You can see this by running `yarn list @types/react`. Running dedupe didn't work so enforcing v18 of types for now.

Already on the latest version of `react-simple-maps`, which doesn't seem like it's maintained anymore...but that's a problem for another day.

Reply:
I was playing around with `react-simple-maps` the other day, we might want to just ditch it at some point. @polemius also had issue with that on mobile, so we'd want a cross-platform solution.

Reply:
@aapeliv @polemius I just made a ticket for this here: #5318 !

@polemius if you want to add a few details to that ticket about what the issues we should be aware of on mobile were (or tell me I can add), that would be great!